manifests/f/feliskio/sleep-timer/2.2.0/feliskio.sleep-timer.yaml

```
@@ -0,0 +1,8 @@
+# Created with komac v2.2.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.6.0.schema.json
+
+PackageIdentifier: feliskio.sleep-timer

```

```suggestion
PackageIdentifier: wotschofsky.sleep-timer
```
Please update file name and path to match this change.